app/templates/org/welcome_page.html

```diff
@@ -1,17 +1,37 @@
 {% extends 'layouts/base.html' %}
 {% import 'macros/dashboard_option.html' as d %}
-
+{% macro dashboard_option(title, endpoint, args={}, description=None, icon=None) %}
```

Comment:
because there's only 1 option, its a bit silly to have it be in a dashboard setup. Could you make it just a link w a description? Also, (nit) could you have it say "Create Organization Page" if the user hasn't created an org yet an "Edit Organization Page" if it has created it?

Reply:
Unsure about the thing re dashboard setup. Right now *it is* a link with description

Also edited text conditional on whether individual has made organization before

![image](https://user-images.githubusercontent.com/4992430/38634726-64f8346a-3d91-11e8-9006-9466f3a404f8.png)

Reply:
thanks!